caravel/assets/javascripts/SqlLab/reducers.js

```diff
@@ -1,6 +1,8 @@
 import shortid from 'shortid';
 import * as actions from './actions';
 import { now } from '../modules/dates';
+import { addToObject, alterInObject, alterInArr, removeFromArr, getFromArr, addToArr }
```

Comment:
interesting tho, eslint complains that trailing comma is not needed here. where does codeclimate get it's js lint rules from? @bkyryliuk